app/src/main/java/com/arygm/quickfix/ui/search/SearchWorkerResult.kt

```diff
@@ -90,158 +117,558 @@ val listOfButtons =
+          "to fly. Its wings are too small to get its fat little body off the ground. The bee, of " +
+          "course, flies anyway because bees don't care what humans think is impossible. Yellow, " +
+          "black. Yellow, black. Yellow, black. Yellow, black. Ooh, black and yellow! Let's shake " +
+          "it up a little. Barry! Breakfast is ready! Coming! Hang on a second. Hello? - Barry? - Adam? " +
+          "- Can you believe this is happening? - I can't. I'll pick you up. Looking sharp. Use the stairs. " +
+          "Your father paid good money for those. Sorry. I'm excited. Here's the graduate. We're " +
+          "very proud of you, son. A perfect report card, all B's. Very proud. Ma! I got a thing " +
+          "going here. - You got lint on your fuzz. - Ow! That's me! - Wave to us! We'll be in row " +
+          "118,000. - Bye! Barry, I told you, stop flying in the house! - Hey, Adam. - Hey, Barry. " +
+          "- Is that fuzz gel? - A little. Special day, graduation. Never thought I'd make it. " +
+          "Three days grade school, three days high school. Those were awkward. Three days college. " +
+          "I'm glad I took a day and hitchhiked around the hive. You did come back different. " +
+          "- Hi, Barry. - Artie, growing a mustache? Looks good. - Hear about Frankie? - Yeah. " +
+          "- You going to the funeral? - No, I'm not going. Everybody knows, sting someone, you die. " +
+          "Don't waste it on a squirrel. Such a hothead. I guess he could have just gotten out of the way. " +
+          "I love this incorporating an amusement park into our day. That's why we don't need vacations. " +
+          "Boy, quite a bit of pomp... under the circumstances."
+  val workerAddress = "Ecublens, VD"
+  val workerRating = 3.8
+  val includedServices =
+      listOf(
+          "Initial Consultation",
+          "Basic Surface Preparation",
+          "Priming of Surfaces",
+          "High-Quality Paint Application",
+          "Two Coats of Paint",
+          "Professional Cleanup")
+  val addonServices =
+      listOf(
+          "Detailed Color Consultation",
+          "Premium paint Upgrade",
+          "Extensive Surface Preparation",
+          "Extra Coats for added Durability",
+          "Power Washing and Deep Cleaning")
+  val rate = 40
+  val tags =
+      listOf(
+          "Exterior Painting",
+          "Interior Painting",
+          "Cabinet Painting",
+          "Licensed & Insured",
+          "Local Worker")
+  val reviews =
+      listOf(
+          "Overall, the work was shit; it’s fair to say that since he just painted the whole mural with " +
+              "his shit. I don’t know what to do; I’ll probably sue him.",
+          "Moha was very punctual and did a great job painting our living room. Highly recommended!",
+          "Moha was very professional and did a fantastic job painting our house",
+          "I wanna marry that man,")
```

Comment:
Again not much to talk about, mehdi is goated but we should remove these eventually and adapt the class with the quickfix backend (I just pushed a PR about it)